realm/src/main/java/io/realm/internal/Table.java

```diff
@@ -711,6 +711,10 @@ private void throwDuplicatePrimaryKeyException(Object value) {
         throw new RealmException("Primary key constraint broken. Value already exists: " + value);
     }
 
+    public void removeIndex(long columnIndex) {
```

Comment:
Yes, since the annotation is called ```@Index``` I feel this is a better name even though we use search index internally.